frontend/src/pages/home/aiFlows/AIFlowHint.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+import {
+  Button,
+  Card,
+  CardBody,
+  CardHeader,
+  Content,
+  Flex,
+  FlexItem,
+} from '@patternfly/react-core';
+import { TimesIcon } from '@patternfly/react-icons';
+import { useBrowserStorage } from '~/components/browserStorage';
+
+type AIFlowHintProps = {
+  title: React.ReactNode;
+  body: React.ReactNode;
+  isDisplayed: boolean;
+  image: React.ReactNode;
+};
+
+const AIFlowHint: React.FC<AIFlowHintProps> = ({ title, body, isDisplayed, image }) => {
+  const [hintHidden, setHintHidden] = useBrowserStorage<boolean>(
+    `odh.dashboard.aiFlow.hint-ai-flows`,
+    false,
+  );
+
+  if (hintHidden || !isDisplayed) {
+    return null;
+  }
+
+  return (
+    <Card data-testid="ai-flow-hint" style={{ borderRadius: 16 }}>
+      <Flex
```

Comment:
Consider using the `Split` layout probably? And you can create a `Flex` to the title, body and image if you want to make it wrappable. In this way, the close button will always be on the right and you don't need to make 2 separate ones here to show on different breakpoints.